docs/manuals/characterization/stark_experiment.rst

```diff
@@ -0,0 +1,225 @@
+the Stark shift with the ``stark_amp`` experiment option.
+Note that ``stark_freq_offset`` should be set as a positive number.
+
+
+.. _stark_frequency_consideration:
+
+Stark tone frequency
+--------------------
+
+As you can see in the equation for :math:`\delta f_S` above,
+:math:`\Delta=0` yields a singular point where :math:`\delta f_S` diverges.
+This corresponds to a Rabi drive, where the qubit is driven on resonance and
+coherent state exchange occurs between :math:`|0\rangle` and :math:`|1\rangle`
+instead of the Stark shift.
+Another frequency that should be avoided for the Stark tone is :math:`\Delta=\alpha` which
+corresponds to the transition from :math:`|1\rangle` to :math:`|2\rangle`.
+In the high power limit, :math:`\Delta = \alpha/2` should also be avoided since
+this causes the direct excitation from :math:`|0\rangle` to :math:`|2\rangle`
+through what is known as a two-photon transition.
+
+The Stark tone frequency must be sufficiently separated from all of these frequencies
+to avoid unwanted state transitions (frequency collisions).
+In reality, the choice of the frequency could be even more complicated
+due to the transition levels of the nearest neighbor qubits.
+The frequency must be carefully chosen to avoid frequency collisions [4]_.
+
+
+.. _stark_channel_consideration:
+
+Stark tone channel
+------------------
+
+It may be necessary to supply a pulse channel to apply the Stark tone.
+In Qiskit Experiments, the Stark experiments usually have an experiment option ``stark_channel``
+to specify this.
+By default, the Stark tone is applied to the same channel as the qubit drive
+with a frequency shift. This frequency shift might update the channel frame,
+which accumulates unwanted phase against the frequency difference between
+the qubit drive :math:`f_0` and Stark tone frequencies :math:`f_S` in addition to
+the qubit Stark shfit :math:`\delta f_s`.
+You can use a dedicated Stark drive channel if available.
+Otherwise, you may want to use a control channel associated with the physical
+drive port of the qubit.
+
+In a typical IBM device using the cross-resonance drive architecture,
+such channel can be identified with your backend as follows:
+
+.. jupyter-execute::
+
+    from qiskit.providers.fake_provider import FakeHanoiV2
+
+    backend = FakeHanoiV2()
+    qubit = 0
+
+    for qpair in backend.coupling_map:
+        if qpair[0] == qubit:
+            break
+
+    print(backend.control_channel(qpair)[0])
+
+This returns a control channel for which the qubit is the control qubit.
+This approach may not work for other device architectures.
+
+
+Characterizing Frequency Shift
+------------------------------
+
+One can experimentally measure :math:`\delta f_S` with the :class:`.StarkRamseyXY` experiment.
+Following pulse sequence illustrates how :math:`\delta f_S` is characterized
+by a variant of the Hahn-echo pulse sequence [5]_.
+
+.. jupyter-execute::
```

Comment:
Hmmm, this is better than it saying "NO THUMB NAIL" but I wonder why it is picking up the qiskit logo and not the image generated by this code block? @coruscating any idea?

![image](https://user-images.githubusercontent.com/1178780/234954633-3dc0caab-123a-407f-b8a4-eceb35694263.png)

Reply:
![image](https://user-images.githubusercontent.com/39517270/235761503-c066de1c-f886-46f9-b24b-7ae84be910ab.png)

Updated. I needed to specify the figure name in the configuration file.
https://nbsphinx.readthedocs.io/en/0.7.1/gallery/thumbnail-from-conf-py.html